marklogic-client-api/src/main/java/com/marklogic/client/datamovement/LineSplitter.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2019 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.marklogic.client.datamovement;
+
+import com.marklogic.client.io.Format;
+import com.marklogic.client.io.StringHandle;
+
+import java.io.*;
+import java.nio.charset.Charset;
+import java.util.stream.Stream;
+
+/**
+ * java doc to do
+ */
```

Comment:
A JavaDoc placeholder that slipped through?

Reply:
I added below txt.
 * The LineSplitter class is used to separate lines in line-delimited JSON, XML or TEXT files.
 * It should also work with gzip-compressed line-delimited files.